index.js

```diff
@@ -181,3 +185,53 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost(prefs.HOSTNAME, function(cookies) {
+    let csrf;
+    let cookieStr = '';
+    cookies.forEach(function(c) {
+      cookieStr += c.name + '=' + c.value + ' ';
+      csrf = c.value;
+    });
+
+    getMe(csrf, cookieStr, function(id) {
+      request({
+        url: 'http://' + prefs.HOSTNAME + '/api/metrics/',
```

Comment:
Looks like you forgot the `?format=json` here.

Might be nice to have an addon-global headers object that contains the Accept, Cookie, and X-CSRFToken headers in it; it would DRY up these request calls nicely. That's optional, though.